cpp/CMakeLists.txt

```
@@ -79,7 +79,7 @@ if(WIN32)
     endif()
 else()
     if(${ARCTICDB_USING_CONDA})
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wextra")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wextra -DGLOG_USE_GLOG_EXPORT")
```

[This is currently used on the feedstock](https://github.com/conda-forge/arcticdb-feedstock/blob/main/recipe/build.sh#L8-L11).

Let's port the comment here and remove it from the feedstock build script then?

```suggestion
        # Required to be able to include headers from glog since glog 0.7
        # See: https://github.com/google/glog/pull/1030
        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wextra -DGLOG_USE_GLOG_EXPORT")
```

Also, `-DDGLOG_USE_GLOG_EXPORT` might be used generally if `glog` is update for the static build which uses vcpkg.